src/components/collapsible/Collapsible.js

```diff
@@ -50,11 +58,26 @@ class Collapsible extends React.Component<Props, State> {
 
     constructor(props: Props) {
         super(props);
+        this.transitionElement = React.createRef();
         this.state = {
             isOpen: props.isOpen,
+            height: '0',
         };
     }
 
+    componentDidMount() {
+        this.updateTransitionElementHeight();
```

Comment:
It seems like this would cause the panel to animate open if `this.props.isOpen` is true initially.